src/TraceEvent/Properties/AssemblyInfo.cs

```diff
@@ -17,3 +17,6 @@
 [assembly: NeutralResourcesLanguage("en")]
 
 // [assembly: InternalsVisibleTo("PerfView")]
+[assembly: AssemblyVersion("1.0.42.0")]
```

Comment:
:memo: There are multiple approaches available which either eliminate the need to manually update these, or dramatically reduce the frequency of changes. If you describe your desired versioning behavior either @AArnott or myself will implement that approach as described or will propose changes in order to make things even simpler. If you are unsure of what you want, we can make recommendations.